src/neuroconv/tools/roiextractors/imagingextractordatachunkiterator.py

```diff
@@ -82,6 +82,17 @@ def __init__(
             progress_bar_options=progress_bar_options,
         )
 
+    def _get_scaled_chunk_shape(self, chunk_mb: float) -> tuple:
```

Comment:
This already exists actually; a recent addition to HDMF that allows the strategy to be overridden

I would mimic the approach of the SI iterator here: https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/tools/spikeinterface/spikeinterfacerecordingdatachunkiterator.py#L74C18-L74C18

Reply:
ah thank you, didn't know about that.